lms/resources/_js_config/file_picker_config.py

```diff
@@ -51,18 +47,13 @@ def blackboard_config(cls, request, application_instance):
     def canvas_config(cls, request, application_instance):
         """Get Canvas files config."""
 
-        enabled = (request.product.family == Product.Family.CANVAS) and (
-            "custom_canvas_course_id" in request.lti_params
-            and application_instance.developer_key is not None
-        )
+        files_enabled = application_instance.settings.get("canvas", "files_enabled")
 
-        auth_url = request.route_url(Canvas.route.oauth2_authorize)
         course_id = request.lti_params.get("custom_canvas_course_id")
-
         config = {
-            "enabled": enabled,
+            "enabled": files_enabled,
             "listFiles": {
-                "authUrl": auth_url,
+                "authUrl": request.route_url(Canvas.route.oauth2_authorize),
                 "path": request.route_path(
                     "canvas_api.courses.files.list", course_id=course_id
                 ),
```

Comment:
This one kind of sticks out to me as it's the only one of three here which includes the details if the files are disabled. Should we make it like the others, or does the frontend expect this?